src/ontology/doid-edit.owl

```diff
@@ -40307,6 +40319,97 @@ AnnotationAssertion(rdfs:label obo:DOID_0060911 "karyomegalic interstitial nephr
+
+https://pubmed.ncbi.nlm.nih.gov/?cmd=link&linkname=pubmed_pubmed&from_uid=26524591")
+AnnotationAssertion(oboInOwl:hasDbXref obo:DOID_0060916 "OMIM:617591")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:DOID_0060916 "PRAAS3")
+AnnotationAssertion(oboInOwl:hasOBONamespace obo:DOID_0060916 "disease_ontology")
+AnnotationAssertion(oboInOwl:id obo:DOID_0060916 "DOID:0060916")
+AnnotationAssertion(rdfs:label obo:DOID_0060916 "Proteasome-associated autoinflammatory syndrome 3"@en)
+SubClassOf(obo:DOID_0060916 obo:DOID_0050737)
+SubClassOf(obo:DOID_0060916 obo:DOID_0060919)
+SubClassOf(obo:DOID_0060916 ObjectSomeValuesFrom(obo:IDO_0000664 obo:GENO_0000930))
+
+# Class: obo:DOID_0060917 (facioscapulohumeral muscular dystrophy 3)
+
+AnnotationAssertion(dc:description obo:DOID_0060917 "Facioscapulohumeral muscular dystrophy-3 (FSHD3) is a digenic muscle disorder characterized by adult onset of proximal muscle weakness affecting the face, neck, scapular muscles, and upper and lower limbs. Muscle involvement is usually asymmetric, and other muscle groups may become involved with progression of the disease")
+AnnotationAssertion(oboInOwl:hasDbXref obo:DOID_0060917 "OMIM:619477")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:DOID_0060917 "FSHD3")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:DOID_0060917 "facioscapulohumeral muscular dystrophy type 3")
+AnnotationAssertion(oboInOwl:hasOBONamespace obo:DOID_0060917 "disease_ontology")
+AnnotationAssertion(oboInOwl:id obo:DOID_0060917 "DOID:0060917")
+AnnotationAssertion(rdfs:label obo:DOID_0060917 "facioscapulohumeral muscular dystrophy 3"@en)
+SubClassOf(obo:DOID_0060917 obo:DOID_11727)
+SubClassOf(obo:DOID_0060917 ObjectSomeValuesFrom(obo:IDO_0000664 obo:GENO_0000930))
+
+# Class: obo:DOID_0060918 (facioscapulohumeral muscular dystrophy 4)
+
+AnnotationAssertion(obo:IAO_0000115 obo:DOID_0060918 "Facioscapulohumeral muscular dystrophy (FSHD) is a skeletal muscle disorder characterized by adult onset of progressive muscle weakness of the face and upper extremity muscles. With disease progression, other muscles also may become affected. There is significant clinical variability and incomplete penetrance.
+
+https://pubmed.ncbi.nlm.nih.gov/27153398/")
+AnnotationAssertion(oboInOwl:hasDbXref obo:DOID_0060918 "OMIM:619478")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:DOID_0060918 "FSHD4")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:DOID_0060918 "facioscapulohumeral muscular dystrophy type 4")
+AnnotationAssertion(oboInOwl:hasOBONamespace obo:DOID_0060918 "disease_ontology")
+AnnotationAssertion(oboInOwl:id obo:DOID_0060918 "DOID:0060918")
+AnnotationAssertion(rdfs:label obo:DOID_0060918 "facioscapulohumeral muscular dystrophy 4"@en)
+SubClassOf(obo:DOID_0060918 obo:DOID_11727)
+
+# Class: obo:DOID_0060919 (autoinflammatory disease)
```

Comment:
recategorized autoinflammatory disease as a subclass of immune system disease according to [NIH's definition](https://www.niams.nih.gov/health-topics/autoinflammatory-diseases/basics/symptoms-causes#:~:text=are%20autoinflammatory%20diseases%3F-,Autoinflammatory%20diseases%20refer%20to%20problems%20with%20the%20immune%20system%2C%20which,blood%20protein%20in%20your%20organs.) as follows: _Autoinflammatory diseases **refer to problems with the immune system**, which usually fights off viruses, bacteria, and infection. The problem causes your immune cells to attack your body by mistake. This can cause swelling that produces fever, rash, joint swelling, or serious buildup of a blood protein in your organs._

Reply:
I think we should avoid adding 'autoinflammatory disease' as a new term as part of these updates _unless it's absolutely necessary._ It is a very broad classification that might already be covered by existing diseases, possibly by 'autoimmune disease' ([DOID:417](https://disease-ontology.org/?id=DOID:417)) or one of its siblings.

Why don't you put on hold any updates to this new grouping disease and instead try to identify the 3 best _existing_ diseases that might be an appropriate parent for the proteasome-associated autoinflammatory syndromes (see [previous comment](https://github.com/DiseaseOntology/HumanDiseaseOntology/pull/1232#pullrequestreview-1567710398)).